src/Bridges/Constraint/bridges/geomean.jl

```diff
@@ -218,9 +218,9 @@ function MOI.get(
 end
 
 function MOI.get(
-    b::GeoMeanBridge{T,F,G},
-    ::MOI.NumberOfConstraints{G,MOI.Nonnegatives},
-)::Int64 where {T,F,G}
+    b::GeoMeanBridge{T,F,G,H},
+    ::MOI.NumberOfConstraints{H,MOI.Nonnegatives},
+)::Int64 where {T,F,G,H}
```

Comment:
These tests found a number of nasty bugs like this one!

Reply:
Nice catch, this is a good validation for the new tests